F-FrontEnd/src/F-intrinsics-types.h

```diff
@@ -282,6 +282,20 @@ typedef enum {
     INTR_ATOMIC_DEFINE,
     INTR_ATOMIC_REF,
 
+    /* F03 intrinsic subroutins */
+    INTR_IS_IOSTAT_END,
+    INTR_IS_IOSTAT_EOR,
+
+    INTR_EXTENDS_TYPE_OF,
+    INTR_SAME_TYPE_AS,
+    INTR_MOVE_ALLOC,
+
+    /* F08 intrinsic subroutines */
+    INTR_COMMOND_ARUGMENT_COUNT,
```

Comment:
Just a typo mistake `COMMOND` should be `COMMAND`. Same for `INTR_GET_COMMOND`, `INTR_GET_COMMOND_ARUGMENT`

Reply:
@clementval Thanks! I'll fix them